Rovers/Rovers.tex

```diff
@@ -0,0 +1,249 @@
+\label{subsec:assessing-technical-skills}
+
+Building a rover, especially one built to URC specifications requires a large team with a diverse set of technical skills. 
+This project will require SPEX to look at and asses what areas are sufficient and what areas need to be developed in order to build a rover. 
+It will also let us know to what spec we are currently capable of building such a rover. 
+
+\subsection{Accessibility}
+\label{subsec:plug-n-play}
+
+The project would be very ambitious. It would require at a minimum a handful (4-5) members each with different areas of expertise. 
+The members would need that knowledge as well as \LaTeX{}, notetaking, and good research practices. 
+This would be to deliver the MVP.
+A larger team would be able to accomplish more.  
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress. 
+The team would take this information, refine it and put it into the technical report along with construction.  
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester. 
+It is also noted that the materials that the team will come across or notes should also be saved in an Google Drive folder for future reference.  
+The team will also present at SPEX design reviews and the weekly checkups at general meetings with the areas the team members are working on that week.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+The largest milestones for the technical report would be the would be draft and completion. 
+The rover itself would include the various CAD models of the rover, software milestones, electrical system completion, and drive train system. 
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+The project is going to require members that are knowledgeable in mechanical design and manufacturing, software engineering, machine learning, electrical engineering, and project management. These members will also need to be experienced in research and design document work. Though this seems simple, finding SPEX members to do this well may be a challenge duye to competition from other projects. 
```

Comment:
>duye
due

Reply:
Are all of these skills absolutely required?
What goals drive the requirement for these skillsets?
Can specific requirements for a skillset be dropped if a goal is reprioritized or dropped from a Primary Objective to a Secondary Objective?